build/Build.cs

```diff
@@ -150,15 +184,16 @@ void ErrorLogger(OutputType type, string text)
       .DependsOn(Test)
       .Executes(() =>
       {
-          //For platform take nuget package description from Directory.Build.Props 
+          //For platform take nuget package description from Directory.Build.Props
+          var version = BuildNumber.IsNullOrEmpty() ? PackageVersion : $"{PackageVersion}.{BuildNumber}";
```

Comment:
why do not define the property ProjectVersion and reuse it in many places to follow the  DRY principle and remove code duplication?